doc/cabal-project-description-file.rst

```diff
@@ -92,7 +92,7 @@ You *must* provide a non-empty list of local packages in your project, filling
 out either a ``packages`` field or an ``optional-packages`` field or both to
 satisfy this requirement.
 
-When ``cabal.project`` doesn't exist, ``cabal-install`` fabricates an ephemeral
+When ``cabal.project`` doesn't exist, :term:`cabal-install:exe:cabal` fabricates an ephemeral
```

Comment:
In other places, `cabal-install` describes the package and `:term:`cabal` the command.